doc/getting_started.rst

```diff
@@ -230,3 +230,6 @@ illustrate the use of ``scikit-learn`` in many different contexts.
 
 The :ref:`tutorials <tutorial_menu>` also contain additional learning
 resources.
+
+In order to pass around sample aligned data such as ``sample_weight``, refer to
```

Comment:
I think it deserves a link at least from getting started, since it'll be a a MUST DO if users are gonna pass around `sample_weight`. Unless you think the whole sample weight business doesn't belong there. I agree that it shouldn't be an extended section, but I think a single sentence with a link to the right doc doesn't hurt.

Reply:
It's only a MUST DO if you want to pass weights in a pipeline or search

Reply:
but we really want users to use pretty much always a `Pipeline`, don't we? Is it really bothering you that it's here? :D  To me it doesn't hurt to have it as the last sentence there, I could remove it if you want, but the more links between topics we have in our docs, the more discoverable material become.

Reply:
It's not _really_ bothering me, it's just unclear to me whether, from the perspective of the newcomer to Scikit-learn, this is information that they would be (a) interested to know, or (b) ready to integrate into their mental model of how Scikit-learn estimators work.